### PR TITLE
Support disconnected network environments

### PR DESCRIPTION
**Pull request by dprince**
This PR adds support for installing the operator in disconnected network environments.

### CODE REVIEW

**Comment by dprince** [comment]
Re the failed CI here. It seems we run 'make bundle' before our operator builds complete in CI. This should resolve the issue: https://github.com/openshift/release/pull/4204

**Comment by dprince** [comment]
Prow jobs are now fixed. The RDO jobs fail with:

Error generating bundle manifests: error resolving image: Get \"https://38.102.83.202:5001/v2/\": http: server gave HTTP response to HTTPS client"

**Comment by raukadah** [comment]
recheck node hold

File: .github/create_bundle.sh [hunk 1]
```diff
@@ -1,69 +1,13 @@
...
-        fi
-    fi
-done
-
-echo "Resulting bundle file images:"
-cat "${CLUSTER_BUNDLE_FILE}" | grep "image:"
-
-# FIXME: display any ENV variables once we have offline support implemented
-#grep -A1 IMAGE_URL_DEFAULT "${CLUSTER_BUNDLE_FILE}"
+USE_IMAGE_DIGESTS=true VERSION=$RELEASE_VERSION IMG=${REGISTRY}/${BASE_IMAGE}:${GITHUB_SHA} make bundle
```
Comment (gibizer) [inline .github/create_bundle.sh:13]: I like that we can simplify the build

**Review by gibizer — COMMENTED** [review]
Looks good to me. I'm not sure if we are OK to land this or it needs other changes to land first. Let me know if we want to land this and I will approve it.

**Review by gibizer — APPROVED** [review]
@dprince confirmed over slack that this has no other dependencies. So as the CI is green and I have noting against the content, I approve it now.

**Pull request merged**